api/openapi.yaml

```diff
@@ -2214,6 +2214,7 @@ components:
           nullable: true
       required:
         - objectType
+        - id
```

Comment:
What does "id" represent here? it is not the database id, right?

Reply:
I think we should call it imageID, to differentiate from the id fields we have on all our resources which represents the database id

Reply:
containerID in that case

Reply:
Thanks for reviewing, @fishkerez ! Agree with you here, it will also make it more consistent with `VMInfo` where we have a `instanceID` 👍